GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -172,6 +237,42 @@ extension CallVisualizer {
 
             return viewController
         }
+
+        private func buildVideoCallViewController(uiConfig: RemoteConfiguration? = nil) -> UIViewController {
+            let coordinator = VideoCallCoodinator(
+                environment: .init(
+                    data: environment.data,
+                    uuid: environment.uuid,
+                    gcd: environment.gcd,
+                    imageViewCache: environment.imageViewCache,
+                    timerProviding: environment.timerProviding,
+                    uiApplication: environment.uiApplication,
+                    date: environment.date,
+                    engagedOperator: environment.engagedOperator,
+                    screenShareHandler: environment.screenShareHandler
+                ),
+                uiConfig: uiConfig,
+                call: .init(
+                    .video(direction: .twoWay),
+                    environment: .init(
+                        audioSession: environment.audiSession,
```

Comment:
Which Audi model do you use 😀
```suggestion
                        audioSession: environment.audioSession,
```

Reply:
haha fixed [3e1a859](https://github.com/salemove/ios-sdk-widgets/pull/494/commits/3e1a8591205f1051a9704059443c98dc92911327)